projects/core/directives/manual-hint/manual-hint.directive.ts

```diff
@@ -16,7 +16,7 @@ export class TuiManualHintDirective extends AbstractTuiHint {
 
     @Input()
     @tuiRequiredSetter()
-    set tuiManualHintShow(show: boolean) {
+    set tuiManualHintShow(show: boolean | null) {
```

Comment:
Let's remove this from this PR like we discussed - we will think about async pipe's nulls later.

Reply:
fixed